proto/penumbra/penumbra/core/component/dex/v1/dex.proto

```diff
@@ -227,6 +227,8 @@ message BatchSwapOutputData {
   TradingPair trading_pair = 8;
   // The starting block height of the epoch for which the batch swap data is valid.
   uint64 epoch_starting_height = 9;
+  // The prefix (epoch, block) of the position where this batch swap occurred.
+  uint64 sct_position_prefix = 10;
```

Comment:
We should deprecate the old field in the protos and remove it from the domain types, the `sct_position_prefix` is what `epoch_starting_height` should have been and we shouldn't carry the old field forward.

Reply:
Done